src/Utils/axiosRetryHandler.ts

```diff
@@ -83,4 +88,13 @@ export function isRetryableError(error: AxiosError): boolean {
       (error.response.status >= 500 && error.response.status <= 599))
   );
 }
-export default axiosRetryHandler;
\ No newline at end of file
+
+export function axiosRetryHandlerWithNotFound(axios: AxiosInstance, axiosRetryOptions: IAxiosRetryOptions): void {
+  axiosRetryHandler(axios, {
+    retries: axiosRetryOptions.retries,
+    waitTimeInMsBetweenRetries: axiosRetryOptions.waitTimeInMsBetweenRetries,
+    shouldRetry: (error) => shouldRetry(error, axiosRetryOptions.retryOn429) || error.response?.status === Constants.notFoundStatusCode,
```

Comment:
Can we move this OR clause ` || error.response?.status === Constants.notFoundStatusCode,` into the `shouldRetry` method? It's confusing having the logic in two different places.

Reply:
its a wrapper around the existing shouldRetry logic